suite/tests/api/ir_aarch64.c

```diff
@@ -94,6 +98,169 @@ test_extend(void *dc)
     test_base_disp_extend(DR_EXTEND_SXTW, false, OPSZ_4, 0);
 }
 
+#define CHECK_INSTR(opcode) \
```

Comment:
I think it would be better to pass in the required arguments to the macro directly, especially `instr`. This makes it easier to see what is going on IMO

Reply:
Yes, it's probably better to pass in as argument. I was in two minds about this.